arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Platform.kt

```diff
@@ -9,10 +9,10 @@ import kotlin.coroutines.CoroutineContext
 import kotlin.coroutines.EmptyCoroutineContext
 import kotlin.coroutines.startCoroutine
 
-internal open class ArrowInternalException(
-  override val message: String =
-    "Arrow-kt internal error. Please let us know and create a ticket at https://github.com/arrow-kt/arrow/issues/new/choose"
-) : RuntimeException(message)
+internal const val ArrowExceptionMessage =
```

Comment:
I need to use this in STM...

Reply:
This can safely be non-internal 👍 

```suggestion
const val ArrowExceptionMessage =
```